addons/account/models/account_move.py

```diff
@@ -2096,8 +2116,7 @@ def compute_tax_repartition_lines_mapping(move_vals):
                 'debit': balance > 0.0 and balance or 0.0,
                 'credit': balance < 0.0 and -balance or 0.0,
             })
-
-            if move_vals['type'] not in ('out_refund', 'in_refund'):
+            if not is_refund:
                 continue
```

Comment:
This condition should come before the loop, directly returning, shouldn't it ? (already before your fix, but while we're at it ...)

If you move it, add a comment at the beginning of the loop indicating that it is only executed for refunds

Reply:
line_vals are updated before this line, refund or not.

Reply:
I read it to quickly, indeed :eyeglasses: